locksmith/.ebextensions/.00_post_migrate.config

```diff
@@ -0,0 +1,17 @@
+files:
+  "/opt/elasticbeanstalk/hooks/appdeploy/post/10_post_migrate.sh":
+    mode: "000755"
+    owner: root
+    group: root
+    content: |
+      #!/usr/bin/env bash
+      if [ -f /tmp/leader_only ]
+      then
+        rm /tmp/leader_only
+        docker exec -e NODE_ENV=production `docker ps --no-trunc -q | head -n 1` npm run db:migrate
+      fi
+
+container_commands:
+  01_migrate:
+    command: "touch /tmp/leader_only"
```

Comment:
Does that means this only runs on a single instance?

Reply:
Yes, this should only run on one of the scaling nodes